src/nxdoc/nuxeo-add-ons/nuxeo-management-rest-api.md

```diff
@@ -0,0 +1,762 @@
+
+```curl
+curl -X POST -u Administrator:Administrator \
+https://localhost:8080/nuxeo/site/management/renditions/pictures/recompute \
+-d 'query=SELECT * FROM Document WHERE ecm:mixinType = "Picture"'
+```
+
+```json
+{
+   "entity-type": "bulkStatus",
+   "commandId": "93baf453-5d34-4bb9-9370-54ec4ffe78b2",
+   "state": "SCHEDULED",
+   "processed": 0,
+   "error": false,
+   "errorCount": 0,
+   "total": 0,
+   "action": "recomputeViews",
+   "username": "system",
+   "submitted": "2019-07-26T12:10:07.567Z",
+   "scrollStart": null,
+   "scrollEnd": null,
+   "processingStart": null,
+   "processingEnd": null,
+   "completed": null,
+   "processingMillis": 0
+}
+```
+
+**Default Query**
+
+If no query is provided, the picture views recomputation is launched on the following default query: `SELECT * FROM Document WHERE ecm:mixinType = 'Picture' AND picture:views/*/title IS NULL`.
+
+#### Recompute Thumbnails
+
+To recompute the thumbnails of the documents matching a given [NXQL]({{page page='nxql'}}) query.
+
+```
+POST https://localhost:8080/nuxeo/site/management/renditions/thumbnails/recompute
+```
+
+**Body Parameters**
+
+| Parameter Name        | Type       | Required | Description                                |
+| --------------------- | ---------- | -------- | ------------------------------------------ |
+| **query**             | **string** |    No    | An NXQL query to select a set of documents.|
+
+
+**Response**
+
+If successful, returns a JSON representation (see below) of a [Bulk Action Status]({{page page='bulk-action-framework'}}#bulk-rest-api).
+
+**Status Codes**
+
+- 200 *OK* - Success.
+
+**Sample**
+
+To recompute the thumbnails for the documents matching a given query:
+
+```curl
+curl -X POST -u Administrator:Administrator \
+https://localhost:8080/nuxeo/site/management/renditions/thumbnails/recompute \
+-d 'query=SELECT * FROM Document WHERE ecm:mixinType = "Thumbnail"'
+```
+
+```json
+{
+   "entity-type": "bulkStatus",
+   "commandId": "0e1e6800-631a-4e04-a47c-241ea7b3596a",
+   "state": "SCHEDULED",
+   "processed": 0,
+   "error": false,
+   "errorCount": 0,
+   "total": 0,
+   "action": "recomputeThumbnails",
+   "username": "system",
+   "submitted": "2019-07-26T12:13:31.361Z",
+   "scrollStart": null,
+   "scrollEnd": null,
+   "processingStart": null,
+   "processingEnd": null,
+   "completed": null,
+   "processingMillis": 0
+}
+```
+
+**Default Query**
+
+If no query is provided, the thumbnail recomputation is launched on the following default query: `SELECT * FROM Document WHERE ecm:mixinType = 'Thumbnail' AND thumb:thumbnail/data IS NULL AND ecm:isVersion = 0 AND ecm:isProxy = 0 AND ecm:isTrashed = 0`.
+
+### Running Status Endpoint
+
+To get all the health check statuses.
+
+```
+GET https://localhost:8080/nuxeo/site/management/runningStatus
+```
+
+**Response**
+
+To retrieve [Bulk Action Status]({{page page='bulk-action-framework'}}#bulk-rest-api) for a given command id.
```

Comment:
That's probably not the expected response description here :)

Reply:
If successful, returns a JSON representation of the health probes information.